CRM/EventsExtras/Hook/BuildForm/EventFee.php

```diff
@@ -16,10 +16,15 @@ public function __construct() {
   }
 
   /**
-  *
-  * @param \CRM_Event_Form_ManageEvent_Fee $form
-  */
-  public function buildForm(CRM_Event_Form_ManageEvent_Fee &$form) {
+   * Hides options on the Event Fee page
+   *
+   * @param string $formName
+   * @param CRM_Event_Form_ManageEvent_Fee $form
+   */
+  public function handle($formName, &$form) {
+    if (!$this->shouldHandle($formName, CRM_Event_Form_ManageEvent_Fee::class)) {
```

Comment:
I think it's better to have the `shouldHandle` function inside each of the classes extending the base class. 
Take for example, we decide in the future that the criteria for this particular hook to be handled is to check if the form class is either of two classes rather than just one class (i.e in that case we will need to do a `in_array` while the check for the other classes remain same, It will be a problem then.
 
You can leave for now though or at least make the `shouldHandle` function  in the base class accept an array of form names to check against.